pkg/gen/nodejs-templates/kind.ts.mustache

```diff
@@ -2,6 +2,7 @@
 // *** Do not edit by hand unless you're certain you know what you are doing! ***
 
 import * as pulumi from "@pulumi/pulumi";
+import { core } from "../..";
```

Comment:
Looks like this isn't always required, but probably doesn't hurt to leave it for simplicity's sake?

Reply:
Yes.